sechub-commons-security-spring/src/test/java/com/mercedesbenz/sechub/spring/security/OAuth2TokenExpirationCalculatorTest.java

```diff
@@ -0,0 +1,176 @@
+// SPDX-License-Identifier: MIT
+package com.mercedesbenz.sechub.spring.security;
+
+import static org.assertj.core.api.Assertions.*;
+import static org.mockito.Mockito.*;
+
+import java.time.Duration;
+import java.time.Instant;
+import java.util.stream.Stream;
+
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.api.extension.ExtensionContext;
+import org.junit.jupiter.params.ParameterizedTest;
+import org.junit.jupiter.params.provider.Arguments;
+import org.junit.jupiter.params.provider.ArgumentsProvider;
+import org.junit.jupiter.params.provider.ArgumentsSource;
+import org.springframework.security.oauth2.core.OAuth2AccessToken;
+
+class OAuth2TokenExpirationCalculatorTest {
+    private static final Duration DEFAULT_EXPIRY = Duration.ofHours(1);
+
+    private OAuth2TokenExpirationCalculator calculatorToTest;
+
+    @BeforeEach
+    void beforeEach() throws Exception {
+        calculatorToTest = new OAuth2TokenExpirationCalculator();
+    }
+
+    @Test
+    void isExpired_throws_illegal_argument_exception_if_now_is_null() {
+        /* prepare */
+        OAuth2OpaqueTokenIntrospectionResponse response = mock();
+
+        /* execute + test */
+        assertThatThrownBy(() -> calculatorToTest.isExpired(response, null)).isInstanceOf(NullPointerException.class).hasMessageContaining("now");
+    }
+
+    @Test
+    void isExpired_throws_illegal_argument_exception_if_now_is_null_() {
```

Comment:
`isExpired_throws_illegal_argument_exception_if_now_is_null_` is not correct
please rename to
`isExpired_throws_illegal_argument_exception_if_response_is_null`